benchmark_test.go

```diff
@@ -0,0 +1,62 @@
+package rollrus
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/benjamindow/rollrus/buffer/diode"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	env      = "test"
+	errorMSG = "test"
+)
+
+func BenchmarkVanillaLogger(b *testing.B) {
+	vanillaLogger := logrus.New()
+	vanillaLogger.Out = ioutil.Discard
+
+	for i := 0; i < b.N; i++ {
+		vanillaLogger.Error(errorMSG)
+	}
+}
+
+func BenchmarkWithChannelBuffer(b *testing.B) {
+	if token == "" {
+		b.Skip("Could not get rollbar token")
+	}
+
+	rollrusLogger := logrus.New()
+	rollrusLogger.Out = ioutil.Discard
+	hook := NewHook(token, env)
```

Comment:
May want to explicitly set the buffer here similar to line 53 to make the benchmark more understandable